datastream-utils/src/main/java/com/linkedin/datastream/common/zk/ZkClient.java

```diff
@@ -347,9 +216,12 @@ public void ensurePath(String path) {
     return (T) _zkSerializer.deserialize(data);
   }
 
+  /**
+   * Get Zk sessions session ID
```

Comment:
fix the javadoc.

Reply:
Removed the method. Not required anymore